instrumentation/undertow/undertow-1.4/build.gradle.kts

```diff
@@ -0,0 +1,37 @@
+plugins {
+    `java-library`
+    id("net.bytebuddy.byte-buddy")
+    id("io.opentelemetry.instrumentation.auto-instrumentation")
+    muzzle
+}
+
+muzzle {
+    pass {
+        group = "io.undertow"
+        module = "undertow-core"
+        versions = "[1.4.0.Final,)"
+        assertInverse = true
+    }
+}
+
+afterEvaluate{
+    io.opentelemetry.instrumentation.gradle.bytebuddy.ByteBuddyPluginConfigurator(project,
+            sourceSets.main.get(),
+            "io.opentelemetry.javaagent.tooling.muzzle.collector.MuzzleCodeGenerationPlugin",
+            project(":javaagent-tooling").configurations["instrumentationMuzzle"] + configurations.runtimeClasspath
+    ).configure()
+}
+
+val versions: Map<String, String> by extra
+
+dependencies {
+    implementation("io.opentelemetry.javaagent.instrumentation:opentelemetry-javaagent-undertow-1.4:${versions["opentelemetry_java_agent"]}")
+    library("io.undertow:undertow-core:2.0.0.Final")
```

Comment:
I think this should be 1.4.0. It is what muzzle check is configred against. The idea is to use the lowest version possible.

Reply:
Works for me 👍  FWIW i copied this part from the [OTEL undertow module](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/undertow-1.4/javaagent/build.gradle#L15)